quinn-proto/src/lib.rs

```diff
@@ -25,7 +25,7 @@ mod range_set;
 mod spaces;
 #[cfg(all(test, feature = "rustls"))]
 mod tests;
-mod transport_parameters;
+pub mod transport_parameters;
```

Comment:
Since we don't need to expose `PreferredAddress`, I think we can just reexport the type rather than the whole module.

Reply:
As mentioned in the PR description, I wanted to avoid exporting `TransportParameters` at the top level, since most API users won't be needing it anyway.

Reply:
I think exporting one type is preferable to exporting one module containing one type regardless--but this turned out to be moot anyway.